src/cxx_supportlib/ServerKit/HttpHeaderParserState.h

```diff
@@ -28,6 +28,8 @@
 
 #include <DataStructures/LString.h>
 #include <Algorithms/Hasher.h>
+#include <ServerKit/llhttp.h>
+#include <ServerKit/HeaderTable.h>
```

Comment:
This include seems to be unnecessary. Maybe you're looking to include some of its side-effects instead?

Reply:
I added it so my LSP could understand the code better.